datameta/api/ui/admin.py

```diff
@@ -85,9 +86,11 @@ def v_admin_put_request(request):
                 site_admin = False,
                 group_admin = newuser_make_admin,
                 pwhash = '!')
-
-        db.add(new_user)
-        db.flush() # We need an ID
+        try:
+            db.add(new_user)
+            db.flush() # We need an ID
+        except IntegrityError:
+            raise errors.get_validation_error("A group with that name already exists.")
```

Comment:
`get_validation_error()` expects a list of messages
```suggestion
            raise errors.get_validation_error(["A group with that name already exists."])
```